spec/fake_braintree/payment_method_spec.rb

```diff
@@ -34,6 +34,26 @@ def token_for(month, year)
   end
 end
 
+describe 'Braintree::PaymentMethod.delete' do
+  it 'successfully deletes a payment method' do
+    token = cc_token # creates card
+
+    result = Braintree::PaymentMethod.delete(token)
+
+    expect(result).to eq true
+
+    expect {
+      Braintree::PaymentMethod.find(token)
+    }.to raise_error(Braintree::NotFoundError)
+  end
+
+  it 'raises an error for a nonexistent payment method' do
+    expect {
+      Braintree::PaymentMethod.delete('foo')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.